### PR TITLE
Instead of giving an ssh alias, now we drop an environment file.

### PR DESCRIPTION
**Pull request by rustyrazorblade**
When sourced, this file brings in a bunch of helpful aliases and bash
functions.

### CODE REVIEW

File: src/main/resources/com/thelastpickle/tlpcluster/configuration/env.sh [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+            ssh $i 'nodetool version' > nodetool/version.txt
+
+            # other OS
+
+            ssh $i 'sudo blockdev --report' > os/blockdev-report.txt
+            ssh $i 'hostname' > os/hostname.txt
+            ssh $i 'cat /proc/meminfo' > os/meminfo.txt
+
+            # configs
+            rsync -ave ssh $i:/etc/cassandra/ conf/
```
Comment (adejanovski) [inline src/main/resources/com/thelastpickle/tlpcluster/configuration/env.sh]: This apparently needs a bit of magic to use the sshConfig file that was generated by tlp-cluster.

Comment (ossarga) [reply src/main/resources/com/thelastpickle/tlpcluster/configuration/env.sh]: I got this to work and made a fix in the following commit.

File: src/main/resources/com/thelastpickle/tlpcluster/configuration/env.sh [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+            ssh $i 'cat /proc/meminfo' > os/meminfo
+            ssh $i 'cat /proc/cpuinfo' > os/cpuinfo
+            ssh $i 'lscpu' > os/lscpu.txt
+
+            # configs
+            rsync $i:/etc/cassandra/ ./conf/
+            rsync $i:/var/log/cassandra/ ./logs/
+
+            # JMX
+            ssh $i -C 'cd provisioning/cassandra; java -cp collector-0.11.1-SNAPSHOT.jar io.prometheus.jmx.JmxScraper service:jmx:rmi:///jndi/rmi://127.0.0.1:7199/jmxrmi' > metrics.jmx
```
Comment (ossarga) [inline src/main/resources/com/thelastpickle/tlpcluster/configuration/env.sh]: Just ran a full end-to-end test. Using this branch I created a cluster then ran the `c-collect-artifacts` command. For some reason the `collector-0.11.1-SNAPSHOT.jar` file is never copied from the resources directory into the provisioning directory. The missing JAR in the provisioning directory causes this line here fail. I'm looking a bit further into it.

Comment (rustyrazorblade) [reply src/main/resources/com/thelastpickle/tlpcluster/configuration/env.sh]: I think there's an issue copying a jar file into resources (or maybe reading it out).  Either way, I addressed this by gzip'ing the JAR into resources, and un'gz-ing when we read it out.  It works

Comment (ossarga) [reply src/main/resources/com/thelastpickle/tlpcluster/configuration/env.sh]: Awesome. Nice fix!

**Pull request merged**